code/game/objects/effects/particle_emitter.dm

```diff
@@ -0,0 +1,14 @@
+///Particle holder for turfs (who cant hold their own emit particles ;_;). Add to a turf with turf..make_stick_to_turf(me)
+/obj/effect/abstract/particle_holder
```

Comment:
bro we already have this exact thing?

Reply:
Holy shit we do wtf

Reply:
I swear I added this completely independently

Reply:
time green wtf

Reply:
Mandella effect code